### DISCUSSION TITLE
Version to use

### DISCUSSION THREAD

**Discussion by basemanabulsi** [hyochan/react-native-iap#1914] [Q&A]
Hello guys, 
Which version should we use?
When I tried to install the package V10.0.3 installed and its the latest, while in the announcement there is nothing about V10.0.0.

Can you help or let us know more about it? 

Thank you, regards

**Comment by andresesfm** [comment]
@basemanabulsi 10.0.0 has some refactorings. mainly using objects as parameters to all methods.

**Comment by andresesfm** [comment]
Just updated readme with latest info

**Comment by linaia** [comment]
You can look at https://github.com/dooboolab/react-native-iap/blob/main/IapExample/src while the documentation is updated
